src/test/java/seedu/address/logic/commands/FindCommandTest.java

```diff
@@ -24,8 +25,8 @@
  * Contains integration tests (interaction with the Model) for {@code FindCommand}.
  */
 public class FindCommandTest {
-    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
-    private Model expectedModel = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs(), new IncomeList());
+    private Model expectedModel = new ModelManager(getTypicalAddressBook(), new UserPrefs(), new IncomeList());
```

Comment:
I dislike this pattern to use `ModelManager`, maybe we can slate a refactoring task later on to clean this up

Reply:
Discussion @ 11 Oct 23: KIV in v1.2b and then crash and burn with implementation to discuss refactoring